lnrpc/peersrpc/peers_server.go

```diff
@@ -248,36 +248,25 @@ func (s *Server) updateFeatures(currentfeatures *lnwire.RawFeatureVector,
 	updates []*UpdateFeatureAction) (*lnwire.RawFeatureVector,
 	*lnrpc.Op, error) {
 
-	ops := &lnrpc.Op{Entity: "features"}
-	raw := currentfeatures.Clone()
+	var (
+		ops    = &lnrpc.Op{Entity: "features"}
+		add    []lnwire.FeatureBit
+		remove []lnwire.FeatureBit
+	)
 
 	for _, update := range updates {
 		bit := lnwire.FeatureBit(update.FeatureBit)
 
 		switch update.Action {
 		case UpdateAction_ADD:
-			if raw.IsSet(bit) {
-				return nil, nil, fmt.Errorf(
-					"invalid add action for bit %v, "+
-						"bit is already set",
-					update.FeatureBit,
-				)
-			}
-			raw.Set(bit)
+			add = append(add, bit)
 			ops.Actions = append(
```

Comment:
This is from an old commit but we should use the bit number whenever `lnwire.Features[bit]` is not ok. If not the cli message is pretty confusing ( If not I can add the fix it i another PR) :pray: 

```
$ lncli peers updatenodeannouncement --feature_bit_add=1002
{
    "ops": [
        {
            "entity": "features",
            "actions": [
                " set"
            ]
        }
    ]
}

```